lib/experian/client.rb

```diff
@@ -49,7 +49,7 @@ def excon_options
     end
 
     def request_uri
-      Experian.net_connect_uri
+      raise "Request uri undefined"
```

Comment:
My intent here is that this client should always be extended to implement this because their different apis have different end points (we only use password reset and precise id but I know the previous implementation of connect check also had a different url).